frontend/packages/ceph-storage-plugin/integration-tests/utils/helpers.ts

```diff
@@ -164,7 +164,7 @@ export const verifyNodeOSDMapping = (
   return filteredOsds.length === 0;
 };
 
-export const hasTaints = (node) => {
+export const hasNoTaints = (node) => {
   return !_.isEmpty(node.spec?.taints);
 };
```

Comment:
```suggestion
export const hasNoTaints = (node) => {
  return _.isEmpty(node.spec?.taints);
};
```
Shouldn't this be?

Reply:
No, the utility is to filter out taints. For ocs nodes table, we show nodes with no taints.